components/molecules/ProfileInfo.js

```diff
@@ -1,17 +1,27 @@
-import ProfileInfoSection from '../atoms/ProfileInfoSection'
+import en from '../../locales/en'
+import fr from '../../locales/fr'
```

Comment:
We shouldn't be importing translations into components anymore, the text should be fed though props to support AEM.

Reply:
Done.